Lombiq.VueJs.Samples/Views/QrCard/Index.cshtml

```diff
@@ -11,6 +11,7 @@
             OverconstrainedError: '@T["Installed cameras are not suitable."]',
             StreamApiNotSupportedError: '@T["The stream API is not supported in this browser."]',
             InsecureContextError: '@T["Camera access is only permitted in secure context. Use HTTPS or localhost rather than HTTP."]',
+            InvalidCard: '@T["This QR code does not represent a business card."]',
```

Comment:
Yes. I can't use `IViewLocalizer` in Vue app.

Reply:
You [**can** use](https://github.com/Lombiq/Orchard-Vue.js/blob/dev/Readme.md#using-vuejs-single-file-components) `IStringLocalizer`, but you would have to move the messages to the `<template>` somehow. Would that work?

Reply:
Currently, I don't see how, but in that case, don't we need defaults in the code?
```javascript
        errorMessages: {
            type: Object,
            default: function (raw) {
                const messages = {
                    NotAllowedError: 'You need to grant camera access permission.',
                    NotFoundError: 'No camera found.',
                    NotSupportedError: 'Secure context required (HTTPS, localhost).',
                    NotReadableError: 'The camera is already in use.',
                    OverconstrainedError: 'Installed cameras are not suitable.',
                    StreamApiNotSupportedError: 'The stream API is not supported in this browser.',
                    InsecureContextError: 'Camera access is only permitted in secure context. Use HTTPS or localhost rather than HTTP.',
                    InvalidCard: 'This QR code does not represent a business card.',
                    Unknown: 'Camera error.',
                };

                return {
                    ...messages,
                    ...(raw || {}),
                };
            },
        },
```